app/generator/content/schedule/get-help-info.js

```diff
@@ -38,7 +38,8 @@ const getHelpInfo = () => {
           '\nFollow us on Twitter @Ruralpay'

```

Comment:
Tests for this file needs moved.

Let's include a test to ensure the unbreakable value is set to `true`.